modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,94 @@
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+As an alternative to the OAuth for GitHub, GitLab, or Bitbucket that is configured by the administrator of your organization's {prod-short} instance, you can apply your Git credentials store as a Secret in your user {orch-namespace}.
+
+When mounting the Secret, a Git configuration file with the path to the mounted Git credentials store is automatically configured and mounted to the {devworkspace} containers at `/etc/gitconfig`.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+
+ifeval::["{project-context}" == "che"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://kubernetes.io/docs/tasks/tools/#kubectl[Install Tools: kubectl].
+endif::[]
+
+ifeval::["{project-context}" != "che"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI].
+endif::[]
+
+* For CLI users: link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using. For Mac users, `base64` is available (as part of `coreutils`) on link:https://formulae.brew.sh/formula/coreutils[homebrew].
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. In your home directory, locate and open your `.git-credentials` file if you already have it. Alternatively, if you do not have this file, save a new `.git-credentials` file, using the link:https://git-scm.com/docs/git-credential-store#_storage_format[Git credentials storage format]. Each credential is stored on its own line in the file:
+
++
+[subs="+quotes,+attributes,+macros"]
+----
+https://__<username>__:__<token>__@__<git_server_hostname>__
+----
+
++
+.A line in a `.git-credentials` file
+====
+`https://trailblazer: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@github.com`
+====
+
+. Select credentials from your `.git-credentials` file for the Secret. Encode the selected credentials to Base64 for the next step.
++
+[TIP]
+====
+CLI users can use the `base64` command as needed:
+
+* To encode all lines in the file:
+** `$ cat .git-credentials | base64 | tr -d '\n'`
+
+* To encode a selected line:
+** `$ echo -n '__<copied_and_pasted_line_from_.git-credentials>__' | base64`
+====
+
+. Create a new {platforms-name} Secret in your user {orch-namespace}.
```

Comment:
@dkwon17, `orch-name` is `Kubernetes` upstream and `OpenShift` downstream, whereas `platforms-name` is `Kubernetes or OpenShift` upstream (i.e., `Create a new Kubernetes or OpenShift Secret`):
```suggestion
. Create a new {orch-name} Secret in your user {orch-namespace}.
```